viewer/src/views/summary/CrashesByYearAverage.js

```diff
@@ -36,12 +36,8 @@ const CrashesByYearAverage = ({ avgData, currentYearData }) => {
       };
     };
 
-    const isDataFetched = !!avgData.length && !!currentYearData.length;
```

Comment:
in addition to testing it, i read through this code a few times and i don't see a reason to be concerned with `isDataFetched`. there could be some sneaky type error lurking here but it seems fine to me 🤷